2020-12-05T09:40:22.0603981+07:00  [INF] Now listening on: "https://localhost:5301" (d826f4b8)
2020-12-05T09:40:22.0676323+07:00  [INF] Now listening on: "http://localhost:5300" (d826f4b8)
2020-12-05T09:40:22.0703918+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-12-05T09:40:22.0727972+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-12-05T09:40:22.0754739+07:00  [INF] Content root path: "D:\BaseProjects\SVN\dotNet5Templates\4.Websites\WebBackend\Web.Backend" (b5d60022)
2020-12-05T09:40:52.4777010+07:00 0HM4OOSED56KT:0000004B [ERR] Get structure of Table '977b21be' failed! (aad82594)
Grpc.Core.RpcException: Status(StatusCode="Internal", Detail="Error starting gRPC call. HttpRequestException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. (192.168.68.182:5100) SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond.", DebugException="System.Net.Http.HttpRequestException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. (192.168.68.182:5100)
 ---> System.Net.Sockets.SocketException (10060): A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond.
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)
   at System.Net.Sockets.Socket.<ConnectAsync>g__WaitForConnectWithCancellation|283_0(AwaitableSocketAsyncEventArgs saea, ValueTask connectTask, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.DefaultConnectAsync(SocketsHttpConnectionContext context, CancellationToken cancellationToken)
   at System.Net.Http.ConnectHelper.ConnectAsync(Func`3 callback, DnsEndPoint endPoint, HttpRequestMessage requestMessage, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at System.Net.Http.ConnectHelper.ConnectAsync(Func`3 callback, DnsEndPoint endPoint, HttpRequestMessage requestMessage, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.ConnectAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.GetHttp2ConnectionAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean async, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)
   at Grpc.Net.Client.Internal.GrpcCall`2.RunCall(HttpRequestMessage request, Nullable`1 timeout)")
   at Web.Backend.Controllers.DynamicForm.DFormController.Index(String fid, String msg) in D:\BaseProjects\SVN\dotNet5Templates\4.Websites\WebBackend\Web.Backend\Controllers\DynamicForm\DFormController.cs:line 43
2020-12-05T09:40:52.7799234+07:00 0HM4OOSED56KT:0000004B [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.ArgumentNullException: Value cannot be null. (Parameter 'source')
   at System.Linq.ThrowHelper.ThrowArgumentNullException(ExceptionArgument argument)
   at System.Linq.Enumerable.Where[TSource](IEnumerable`1 source, Func`2 predicate)
   at AspNetCore.Views_DForm_Index.ExecuteAsync() in D:\BaseProjects\SVN\dotNet5Templates\4.Websites\WebBackend\Web.Backend\Views\DForm\Index.cshtml:line 7
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.RenderPageCoreAsync(IRazorPage page, ViewContext context)
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.RenderPageAsync(IRazorPage page, ViewContext context, Boolean invokeViewStarts)
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.RenderAsync(ViewContext context)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewExecutor.ExecuteAsync(ViewContext viewContext, String contentType, Nullable`1 statusCode)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewExecutor.ExecuteAsync(ViewContext viewContext, String contentType, Nullable`1 statusCode)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewExecutor.ExecuteAsync(ActionContext actionContext, IView view, ViewDataDictionary viewData, ITempDataDictionary tempData, String contentType, Nullable`1 statusCode)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewResultExecutor.ExecuteAsync(ActionContext context, ViewResult result)
   at Microsoft.AspNetCore.Mvc.ViewResult.ExecuteResultAsync(ActionContext context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeResultAsync>g__Logged|21_0(ResourceInvoker invoker, IActionResult result)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResultFilterAsync>g__Awaited|29_0[TFilter,TFilterAsync](ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResultExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.ResultNext[TFilter,TFilterAsync](State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeResultFilters()
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-12-05T09:41:54.7194613+07:00  [INF] Now listening on: "https://localhost:5301" (d826f4b8)
2020-12-05T09:41:54.7265082+07:00  [INF] Now listening on: "http://localhost:5300" (d826f4b8)
2020-12-05T09:41:54.7297841+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-12-05T09:41:54.7329414+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-12-05T09:41:54.7357505+07:00  [INF] Content root path: "D:\BaseProjects\SVN\dotNet5Templates\4.Websites\WebBackend\Web.Backend" (b5d60022)
2020-12-05T09:42:02.0669674+07:00 0HM4OOTA0F986:00000003 [ERR] Get structure of Table '977b21be' failed! (aad82594)
Grpc.Core.RpcException: Status(StatusCode="Internal", Detail="Error starting gRPC call. HttpRequestException: No connection could be made because the target machine actively refused it. (192.168.68.113:5100) SocketException: No connection could be made because the target machine actively refused it.", DebugException="System.Net.Http.HttpRequestException: No connection could be made because the target machine actively refused it. (192.168.68.113:5100)
 ---> System.Net.Sockets.SocketException (10061): No connection could be made because the target machine actively refused it.
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)
   at System.Net.Sockets.Socket.<ConnectAsync>g__WaitForConnectWithCancellation|283_0(AwaitableSocketAsyncEventArgs saea, ValueTask connectTask, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.DefaultConnectAsync(SocketsHttpConnectionContext context, CancellationToken cancellationToken)
   at System.Net.Http.ConnectHelper.ConnectAsync(Func`3 callback, DnsEndPoint endPoint, HttpRequestMessage requestMessage, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at System.Net.Http.ConnectHelper.ConnectAsync(Func`3 callback, DnsEndPoint endPoint, HttpRequestMessage requestMessage, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.ConnectAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.GetHttp2ConnectionAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean async, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)
   at Grpc.Net.Client.Internal.GrpcCall`2.RunCall(HttpRequestMessage request, Nullable`1 timeout)")
   at Web.Backend.Controllers.DynamicForm.DFormController.Index(String fid, String msg) in D:\BaseProjects\SVN\dotNet5Templates\4.Websites\WebBackend\Web.Backend\Controllers\DynamicForm\DFormController.cs:line 43
2020-12-05T09:42:41.9211152+07:00  [INF] Now listening on: "https://localhost:5301" (d826f4b8)
2020-12-05T09:42:41.9289573+07:00  [INF] Now listening on: "http://localhost:5300" (d826f4b8)
2020-12-05T09:42:41.9316273+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-12-05T09:42:41.9338100+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-12-05T09:42:41.9359780+07:00  [INF] Content root path: "D:\BaseProjects\SVN\dotNet5Templates\4.Websites\WebBackend\Web.Backend" (b5d60022)
2020-12-05T09:42:48.1692941+07:00 0HM4OOTO1IHF0:00000003 [ERR] Get structure of Table '977b21be' failed! (aad82594)
Grpc.Core.RpcException: Status(StatusCode="Internal", Detail="Error starting gRPC call. HttpRequestException: No connection could be made because the target machine actively refused it. (192.168.68.113:5100) SocketException: No connection could be made because the target machine actively refused it.", DebugException="System.Net.Http.HttpRequestException: No connection could be made because the target machine actively refused it. (192.168.68.113:5100)
 ---> System.Net.Sockets.SocketException (10061): No connection could be made because the target machine actively refused it.
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)
   at System.Net.Sockets.Socket.<ConnectAsync>g__WaitForConnectWithCancellation|283_0(AwaitableSocketAsyncEventArgs saea, ValueTask connectTask, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.DefaultConnectAsync(SocketsHttpConnectionContext context, CancellationToken cancellationToken)
   at System.Net.Http.ConnectHelper.ConnectAsync(Func`3 callback, DnsEndPoint endPoint, HttpRequestMessage requestMessage, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at System.Net.Http.ConnectHelper.ConnectAsync(Func`3 callback, DnsEndPoint endPoint, HttpRequestMessage requestMessage, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.ConnectAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.GetHttp2ConnectionAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean async, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)
   at Grpc.Net.Client.Internal.GrpcCall`2.RunCall(HttpRequestMessage request, Nullable`1 timeout)")
   at Web.Backend.Controllers.DynamicForm.DFormController.Index(String fid, String msg) in D:\BaseProjects\SVN\dotNet5Templates\4.Websites\WebBackend\Web.Backend\Controllers\DynamicForm\DFormController.cs:line 43
2020-12-05T09:44:21.6589920+07:00 0HM4OOTO1IHF0:00000015 [ERR] Get datatable of Table '35bc6492' failed!. ERROR: Value cannot be null. (Parameter 'source') (6c98c488)
System.ArgumentNullException: Value cannot be null. (Parameter 'source')
   at System.Linq.ThrowHelper.ThrowArgumentNullException(ExceptionArgument argument)
   at System.Linq.Enumerable.Select[TSource,TResult](IEnumerable`1 source, Func`2 selector)
   at Web.Backend.Controllers.DynamicForm.DFormController.GetDatatable(DFormListViewModel model) in D:\BaseProjects\SVN\dotNet5Templates\4.Websites\WebBackend\Web.Backend\Controllers\DynamicForm\DFormController.cs:line 97
2020-12-05T09:44:23.8213441+07:00 0HM4OOTO1IHF0:00000019 [ERR] Get datatable of Table '34a361f0' failed!. ERROR: Value cannot be null. (Parameter 'source') (3ad27f78)
System.ArgumentNullException: Value cannot be null. (Parameter 'source')
   at System.Linq.ThrowHelper.ThrowArgumentNullException(ExceptionArgument argument)
   at System.Linq.Enumerable.Select[TSource,TResult](IEnumerable`1 source, Func`2 selector)
   at Web.Backend.Controllers.DynamicForm.DFormController.GetDatatable(DFormListViewModel model) in D:\BaseProjects\SVN\dotNet5Templates\4.Websites\WebBackend\Web.Backend\Controllers\DynamicForm\DFormController.cs:line 97
2020-12-05T09:44:26.0992094+07:00 0HM4OOTO1IHF0:0000001D [ERR] Get datatable of Table '35bc6492' failed!. ERROR: Value cannot be null. (Parameter 'source') (6c98c488)
System.ArgumentNullException: Value cannot be null. (Parameter 'source')
   at System.Linq.ThrowHelper.ThrowArgumentNullException(ExceptionArgument argument)
   at System.Linq.Enumerable.Select[TSource,TResult](IEnumerable`1 source, Func`2 selector)
   at Web.Backend.Controllers.DynamicForm.DFormController.GetDatatable(DFormListViewModel model) in D:\BaseProjects\SVN\dotNet5Templates\4.Websites\WebBackend\Web.Backend\Controllers\DynamicForm\DFormController.cs:line 97
2020-12-05T09:44:30.0075313+07:00 0HM4OOTO1IHF0:00000021 [ERR] Get datatable of Table '34a361f0' failed!. ERROR: Value cannot be null. (Parameter 'source') (3ad27f78)
System.ArgumentNullException: Value cannot be null. (Parameter 'source')
   at System.Linq.ThrowHelper.ThrowArgumentNullException(ExceptionArgument argument)
   at System.Linq.Enumerable.Select[TSource,TResult](IEnumerable`1 source, Func`2 selector)
   at Web.Backend.Controllers.DynamicForm.DFormController.GetDatatable(DFormListViewModel model) in D:\BaseProjects\SVN\dotNet5Templates\4.Websites\WebBackend\Web.Backend\Controllers\DynamicForm\DFormController.cs:line 97
2020-12-05T09:44:35.8655330+07:00 0HM4OOTO1IHF0:00000025 [ERR] Get datatable of Table '35bc6492' failed!. ERROR: Value cannot be null. (Parameter 'source') (6c98c488)
System.ArgumentNullException: Value cannot be null. (Parameter 'source')
   at System.Linq.ThrowHelper.ThrowArgumentNullException(ExceptionArgument argument)
   at System.Linq.Enumerable.Select[TSource,TResult](IEnumerable`1 source, Func`2 selector)
   at Web.Backend.Controllers.DynamicForm.DFormController.GetDatatable(DFormListViewModel model) in D:\BaseProjects\SVN\dotNet5Templates\4.Websites\WebBackend\Web.Backend\Controllers\DynamicForm\DFormController.cs:line 97
2020-12-05T09:44:39.6737236+07:00 0HM4OOTO1IHF0:0000002D [ERR] Get datatable of Table '34a361f0' failed!. ERROR: Value cannot be null. (Parameter 'source') (3ad27f78)
System.ArgumentNullException: Value cannot be null. (Parameter 'source')
   at System.Linq.ThrowHelper.ThrowArgumentNullException(ExceptionArgument argument)
   at System.Linq.Enumerable.Select[TSource,TResult](IEnumerable`1 source, Func`2 selector)
   at Web.Backend.Controllers.DynamicForm.DFormController.GetDatatable(DFormListViewModel model) in D:\BaseProjects\SVN\dotNet5Templates\4.Websites\WebBackend\Web.Backend\Controllers\DynamicForm\DFormController.cs:line 97
2020-12-05T10:10:57.7469269+07:00 0HM4OOTO1IHF1:00000009 [ERR] Get datatable of Table '34a361f0' failed!. ERROR: Value cannot be null. (Parameter 'source') (3ad27f78)
System.ArgumentNullException: Value cannot be null. (Parameter 'source')
   at System.Linq.ThrowHelper.ThrowArgumentNullException(ExceptionArgument argument)
   at System.Linq.Enumerable.Select[TSource,TResult](IEnumerable`1 source, Func`2 selector)
   at Web.Backend.Controllers.DynamicForm.DFormController.GetDatatable(DFormListViewModel model) in D:\BaseProjects\SVN\dotNet5Templates\4.Websites\WebBackend\Web.Backend\Controllers\DynamicForm\DFormController.cs:line 97
2020-12-05T11:59:52.1231755+07:00  [INF] Now listening on: "https://localhost:5301" (d826f4b8)
2020-12-05T11:59:52.1332637+07:00  [INF] Now listening on: "http://localhost:5300" (d826f4b8)
2020-12-05T11:59:52.1386656+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-12-05T11:59:52.1413293+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-12-05T11:59:52.1440766+07:00  [INF] Content root path: "D:\BaseProjects\SVN\dotNet5Templates\4.Websites\WebBackend\Web.Backend" (b5d60022)
2020-12-05T12:10:39.3403617+07:00  [INF] Now listening on: "https://localhost:5301" (d826f4b8)
2020-12-05T12:10:39.3477980+07:00  [INF] Now listening on: "http://localhost:5300" (d826f4b8)
2020-12-05T12:10:39.3515392+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-12-05T12:10:39.3558264+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-12-05T12:10:39.3584485+07:00  [INF] Content root path: "D:\BaseProjects\SVN\dotNet5Templates\4.Websites\WebBackend\Web.Backend" (b5d60022)
